data/abilities/credential-access/9179709f-23f6-482f-9a1e-025cea20bd6e.yml

```diff
@@ -0,0 +1,21 @@
+---
+- id: 9179709f-23f6-482f-9a1e-025cea20bd6e
+  name: Powerkatz (XOR'd Payload)
+  description: Use Invoke-Mimikatz
+  tactic: discovery
+  technique:
+    attack_id: T1018
+    name: Remote System Discovery
+  platforms:
+    windows:
+      psh:
```

Comment:
how is this different than the invoke-mimikatz ability we currently have?

Reply:
This ability doesn't pull the mimikatz Powershell script from the public GitHub page and this one relies on the automatic payload encoder. However, I'm cool with just editing the other mimikatz ability rather than including this one.

Reply:
yeah, let's go with the existing one, which pulls from the internet. you can remove the XOR'd file as well.

Reply:
Will do

Reply:
One thing that I just thought about, if we put the 'requirements' relationship into existing lateral movement ability yaml than we will break existing worm adversaries that use username/password facts from any ability or source other than our mimikatz ability. So even if they load in a username and password from the UI, we don't have an interface to allow people to load fact relationships. They would have to manually add them to the sqlite database or through the fact yaml. Because of this, I took out the changes to our existing lat movement abilities. I can put them back in if you like, but this PR has fact relationship examples that people can reference.